services/121-service/src/registration/registrations.controller.ts

```diff
@@ -200,25 +200,23 @@ export class RegistrationsController {
 
   @Permissions(PermissionEnum.RegistrationCREATE)
   @ApiOperation({
-    summary: 'Import set of registered PAs, from JSON only used in testing ATM',
+    summary: 'Import set of registered PAs from JSON ',
   })
   @ApiParam({ name: 'programId', required: true, type: 'integer' })
-  @Post('programs/:programId/registrations/import-registrations-cypress')
```

Comment:
1. I don't think you can just change this existing endpoint. That is used with its old name in the Cypress tests.
2. Admittedly I'm not sure of the exact refinement here, but I would expect this endpoint to be in the espocrm module, and to be of a similar fashion as the updateRegistrations and deleteRegistrations endpoints already in there. So from the espocrm.controller it is fine to call the registration.service for the actual business logic.
3. Related to the above point, I would expect this item to also be tested by actually creating a PA in Espo and see if it creates a registration via the webhook. I can imagine you cannot do this yourselves, as you need access to Espo, so we should team up to do that. But it is necessary for this item to be completed, so please set up what you need.

@TatevAccenture @kaspars-grinbergs-accenture

Reply:
Thanks @jannisvisser, will look into this. Also saw Tijs comment regarding #2 on Teams.

Reply:
I'm resolving this conversation , as we had after discussions and decisions for this